Cython/Utility/ImportExport.c

```diff
@@ -59,7 +59,7 @@ static PyObject *__Pyx__ImportDottedModule_Error(PyObject *name, PyObject *parts
 #if PY_MAJOR_VERSION >= 3
 static PyObject *__Pyx__ImportDottedModule_Lookup(PyObject *name) {
     PyObject *imported_module;
-#if PY_VERSION_HEX < 0x030700A1
+#if PY_VERSION_HEX < 0x030700A1 || CYTHON_COMPILING_IN_PYPY
```

Comment:
This is part of the upcoming PyPy 7.3.4 release
```suggestion
#if PY_VERSION_HEX < 0x030700A1 || (CYTHON_COMPILING_IN_PYPY && PYPY_VERSION_NUM  < 0x07030400
```

Reply:
Yes I was not aware, that it is already implemented. I updated the commit accordingly.